README.md

```diff
@@ -65,6 +65,7 @@ parameters:
     bin_dir: "./vendor/bin"
     git_dir: "."
     stop_on_failure: false
+    ignore_unstaged_changes: false
```

Comment:
Continuing https://github.com/phpro/grumphp/issues/100#issuecomment-179442140
Since you implement great safety guard with calling stash-pop on `register_shutdown_function` when needed, shouldn't this be true by default?
The only problem for setting this as default was making the changes disappear if sth goes wrong - but now it won't happen.